main_app/templates/main_app/login.html

```diff
@@ -58,22 +130,18 @@ <h1 class="display-4" style="text-align: center; color: black; font-family: 'Ban
                         Registered
                         Account</b></p>
 
-                <form method="POST" style="background-color: white;">
+                <form method="POST" class="form-style">
                     <img src="{% static 'Images/signup-image.jpg' %}" alt="Log in"
                         style="margin-left: 30%; height: 35%; width: 35%;">
                     {% csrf_token %}
                     {{form.as_p}}
+                    
                     <br>
-                    <div style="display: flex; align-items: center; justify-content: center;">
-                        <button style="border-radius: 25px;" class="btn btn-success" type="submit"><b>Sign
-                                In</b></button>
+                    
+                    <div class="login">
```

Comment:
Please fix the space above `WELCOME` for the devices with smaller widths and heights.
![image](https://user-images.githubusercontent.com/53635792/111614775-ffa2b780-8805-11eb-8d1c-81e426cc84e4.png)